ui/app/templates/vault/cluster/replication-dr-promote/index.hbs

```diff
@@ -3,6 +3,7 @@
     <ReplicationPage @model={{model}} as |Page|>
       <Page.header
         @showTabs={{true}}
+        @pageType="manage"
```

Comment:
used to determine on manage page or not, and if you are change the link the breadcrumb uses.